src/main/java/com/example/WMS/services/IncomingGoodsServices.java

```diff
@@ -0,0 +1,9 @@
+package com.example.WMS.services;
+
+import com.example.WMS.dtos.*;
```

Comment:
Don't use asterisk for imports, only import what's required. 
You can make these changes in intellij configuration